site/documentation/content/api/Metrics.md

```diff
@@ -118,5 +118,5 @@ For an incoming message of size 10KB, it is reported as 12KB.
 
 | Metric                             | Type                | Tags                                                     | Description |
 | ---------------------------------- | ------------------- | -------------------------------------------------------- | ----------- |
-| *hono.commands.received*           | Timer               | *host*, *component-type*, *component-name*, *tenant*, *type*, *status*, *direction* | The time it took to process a message conveying a command that could not be forwarded to a protocol adapter. |
-| *hono.commands.payload*            | DistributionSummary | *host*, *component-type*, *component-name*, *tenant*, *type*, *status*, *direction* | The number of bytes conveyed in the payload of a command message that could not be forwarded to a protocol adapter. |
+| *hono.command.payload*             | DistributionSummary | *host*, *component-type*, *component-name*, *tenant*, *type*, *status*, *direction* | The number of bytes conveyed in the payload of a command message that could not be forwarded to a protocol adapter. |
+| *hono.command.processing.duration* | Timer               | *host*, *component-type*, *component-name*, *tenant*, *type*, *status*, *direction* | The time it took to process a message conveying a command that could not be forwarded to a protocol adapter. |
```

Comment:
Are these metrics really about messages that **can not** be forwarded? If so, wouldn't it be easier to understand, if the meter name reflected this somehow, e.g. *hono.failed.command.payload* etc?

Reply:
In earlier Hono versions, when there was no Command Router yet, the `hono.command` metrics only got reported in the protocol adapter. With the introduction of the Command Router component, the reporting of successful commands has been kept in the protocol adapter and only commands that couldn't get forwarded to a protocol adapter get reported in the Command Router.
This way, the overall number of `hono.command` metric items reflects the number of received commands.

I've amended the commit here, adding an explanation in the `Command Router` chapter:
```
Command messages get first received by the Command Router component and then get forwarded
to the matching protocol adapter instance, where the command will be reflected in the corresponding
protocol adapter metrics with the *hono.command* prefix.
If the command could not be forwarded to a protocol adapter, the Command Router will report the
command in its metrics, as listed below.
```

> If so, wouldn't it be easier to understand, if the meter name reflected this somehow, e.g. *hono.failed.command.payload* etc?

Then we would have to add *hono.failed.command.payload* also to the protocol adapter metrics (for cases where command processing fails there). But the failed commands can already be determined via the `status` tag, so I don't see the need for such a new metric.

In general, when seeing the `hono.command.processing.duration` metric independent of the Hono component it is associated with, the values there can be seen as misleading. because they encompass neither the processing time inside the Command Router, nor the time spent in transfer between Command Router and protocol adapter.
I think a separate Hono issue should be created for that.